apm-sniffer/apm-sdk-plugin/mysql-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v6/define/PreparedStatementInstrumentation.java

```diff
@@ -53,10 +56,19 @@
         return new InstanceMethodsInterceptPoint[] {
             new InstanceMethodsInterceptPoint() {
                 @Override public ElementMatcher<MethodDescription> getMethodsMatcher() {
-                    return named("execute")
+                    ElementMatcher.Junction<MethodDescription> matcher = named("execute")
                         .or(named("executeQuery"))
                         .or(named("executeUpdate"))
                         .or(named("executeLargeUpdate"));
+                    if (Config.Plugin.MySQL.TRACE_SQL_PARAMETERS) {
+                        for (String setter : PS_SETTERS) {
+                            matcher = matcher.or(named(setter));
+                        }
+                        for (String setter : PS_IGNORED_SETTERS) {
+                            matcher = matcher.or(named(setter));
+                        }
+                    }
+                    return matcher;
```

Comment:
Same to
>I still have performance concerns for these naming match. You should set a explicit instrumentation for get parameter, rather than using string match.
Instrumentation name match filter happens in startup amd class loading stage.